modules/ROOT/pages/endpoint-https-receive.adoc

```diff
@@ -6,31 +6,39 @@ Configure HTTP and HTTPS receive endpoints to receive application messages (JSON
 |===
 |Name |Description |Required
 |Endpoint Name
-|Name used to refer to the endpoint. The name is auto populated, but you can overwrite it based on your organization’s naming conventions for consistency.
+|The name used to identify the endpoint. The `Endpoint Name` field is generated automatically, but can be modified manually.
 |Yes
 
-|Description
-|Freeform description of the endpoint.
-|No
+| Owner
+| The trading partner or host that owns the endpoint.
+| Yes
+
+| Protocol
+| The protocol selected in the `Protocol` field of the *New endpoint: Receive from partners* window. The value in this field cannot be modified.
+| Yes
+
+| Description
+| The user-defined field for describing the purpose and use of the endpoint.
+| No
 
 |URL
 a|The HTTP or HTTPS address where you will receive outbound source messages from backend systems.
 
-When deploying to CloudHub, the URL is automatically generated when you deploy a message flow that uses this endpoint. You can view the URL by selecting the receiving endpoint from your deployed message flow.
+When deploying to CloudHub, the URL is automatically generated when you deploy a message flow that uses this endpoint. To view the URL, you can select the receiving endpoint from your deployed message flow.
 
-If you are deploying to on-premises Mule with a firewall and load balancer, work with your hosting or network security team to obtain the externally accessible DNS or name of the host where the Mule server is running, and replace `{runtime_host}` in the URL with the DNS name of your host. For example, if your DNS host name is `b2b.mycompany.com`, the receiver URL is `+https://b2b.mycompany.com/base-path/+`.
+If you are deploying to on-premises Mule server with a firewall and load balancer, work with your hosting or network security team to obtain the externally accessible DNS or name of the host where the Mule server is running, and replace `{runtime_host}` in the URL with the DNS name of your host. For example, if your DNS host name is `b2b.mycompany.com`, the receiver URL is `+https://b2b.mycompany.com/base-path/+`.
 |
 
 |Connection idle timeout
-|Connection idle timeout in milliseconds. The default is `3000`.
+|The connection idle timeout period in milliseconds. The value in this field is set to `3000` by default.
 |
 
 |Base path
-|Base path for the HTTP listener
+|The base path for the HTTP listener
```

Comment:
Please check this table column's punctuation GLOBALLY in the PR.

As with a list, if even one entry contains a complete sentence (ending with a period), then all entries must end with a period.

If you can write around situations like this, that is great!

Peer list entries, steps, and table column entries should be parallel, if possible.

Reply:
Done!